docs/docs/1.2.3/policies/virtual-outbound.md

```diff
@@ -0,0 +1,254 @@
+# Virtual Outbounds
+
+This policy enables users to create ways to reach a specific set of dataplanes with a `hostname:port` generated from the dataplanes tags.
+
+Possible use cases are:
+
+1) Preserving hostnames when migrating to service mesh (This enables migration without modifying app configuration).
+2) Providing multiple hostnames for reaching the same service (when renaming or for usability).
+3) Providing specific routes (reach a specific pod in a service when using StatefulSets, add an url to reach a specific version of a service).
+4) Expose multiple inbounds on different ports.
+
```

Comment:
```suggestion

Note that complex virtual outbounds do not work for cross-zone traffic. This is because only service tags are propagated across zones. 
```
(moved up from old Collisions section)